<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SphinxTestConfigurationWidget</class>
 <widget class="QFrame" name="SphinxTestConfigurationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>181</height>
   </rect>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tag:</string>
       </property>
       <property name="buddy">
        <cstring>leTag</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leTag"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sphinx input files</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbModelDir">
        <property name="text">
         <string>Model directory:</string>
        </property>
        <property name="buddy">
         <cstring>urModelDir</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="urModelDir"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbSphinxDict">
        <property name="text">
         <string>Dictionary:</string>
        </property>
        <property name="buddy">
         <cstring>urSphinxDict</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="urSphinxDict"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbSphinxGrammar">
        <property name="text">
         <string>Grammar</string>
        </property>
        <property name="buddy">
         <cstring>urSphinxGrammar</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="KUrlRequester" name="urSphinxGrammar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string comment="Test the model">Test configuration</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lbTestPrompts">
        <property name="text">
         <string>Test-prompts:</string>
        </property>
        <property name="buddy">
         <cstring>urTestPrompts</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KUrlRequester" name="urTestPrompts"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbTestPromptsBasePath">
        <property name="text">
         <string>Test-prompts base path:</string>
        </property>
        <property name="buddy">
         <cstring>urTestPromptsBasePath</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KUrlRequester" name="urTestPromptsBasePath"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbSampleRate">
        <property name="text">
         <string>Samplerate:</string>
        </property>
        <property name="buddy">
         <cstring>sbSampleRate</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sbSampleRate">
        <property name="minimum">
         <number>8000</number>
        </property>
        <property name="maximum">
         <number>128000</number>
        </property>
        <property name="value">
         <number>16000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pbRemove">
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QWidget</extends>
   <header>kurlrequester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
